src/sentry/conf/server.py

```diff
@@ -1260,6 +1260,6 @@ def get_raven_config():
 
 SOUTH_TESTS_MIGRATE = os.environ.get('SOUTH_TESTS_MIGRATE', '0') == '1'
 
-PUBSUB_ENABLED = False
-PUBSUB_CONNECTION = ('example', 1234, 0)
-PUBSUB_SAMPLING = {'requests': 0.1}
+PUBSUB_ENABLED = True
+PUBSUB_CONNECTION = ('hoff', 6379, 0)
```

Comment:
Yeah, we'll absolutely not configure this here, and we'll do it in `getsentry`. :)